package/com.unity.formats.usd/Tests/Editor/UsdPrefabTests.cs

```diff
@@ -33,7 +31,7 @@ public class Reload : UsdPrefabTests
             [Ignore("USDU-276")]
             public void UsdPrefab_ForceReload_Succeed(MaterialImportMode mode)
             {
-                ImportAsPrefab(k_USDZGUID);
+                ImportAsPrefab(TestDataGuids.PrimType.ComponentPayloadUsd);
```

Comment:
This should be 'TestDataGuids.PrimType.ComponentPayloadUsd**z**' so that it's super clear on file type

Reply:
Fixed!